tortoise/tracer.go

```diff
@@ -119,8 +119,6 @@ const (
 	traceTally
 	traceBlock
 	traceHare
-	traceActiveset
-	traceResults
```

Comment:
@dshulyak Is it OK to remove unused ones (its iota so the consts below will change values)?

Reply:
It should only be a problem if the "enum" is persisted somewhere, otherwise the concrete values of the options don't really matter.

I think we do further below in `(e *eventEnum) Decode` which is called from `RunTrace` that parses a file.

Reply:
By default the tracer is disabled, so it might still be OK to drop those enums since I think we only use it for testing and will always use the same version of a node to generate an parse a trace, but correct me if I'm wrong @dshulyak

Reply:
i think i forgot to remove this traces when i removed related methods. it is fine to drop them